### ISSUE TITLE
Incompatible and old pip packages.Many operations are not run stably

### ISSUE THREAD

**Issue by Ildemselcuk** [flow-project/flow#1033]
Hello,

Most of the pip packages in Requirements.txt are minus versions and they are incompatible with each other, so can you send the updated requirements.txt file or pip package versions.Many operations are not run stably for this reason.

```
gym==0.14.0
numpy==1.18.4
scipy==1.1.0
lxml==4.4.1
pyprind==2.11.2
nose2==0.8.0
six==1.11.0
path.py
joblib==0.10.3
python-dateutil==2.7.3
cached_property
pyglet==1.3.2
matplotlib==3.1.0
imutils==0.5.1
numpydoc
ray==0.8.0
opencv-python
dill
lz4
setproctitle
psutil
opencv-python
boto3==1.10.45
redis~=2.10.6
pandas==0.24.2
plotly==2.4.0
tabulate
tensorflow==1.15.2
awscli==1.16.309
torch==1.4.0
pytz
tensorboardX
```
Best Regards,

**Comment by philippwulff** [comment]
[This](https://github.com/flow-project/flow/issues/1030) appears to be the same issue. Please provide updated dependencies.

**Comment by amanpatel123** [comment]
Is this issue fixed?

**Comment by philippwulff** [comment]
No, it is not fixed. Personally, I just gave up on trying to use this project...

**Comment by arashtavakoli** [comment]
I wish they have responded to all these issues and wouldn't just introduce a package and call it a day!

**Comment by HappyDay-design** [comment]
Hello all

I installed Flow with repleacing the "redis~=2.10.6" by "redis".

But when I ran tutorial 3 and 4, it had error.

AttributeError: 'numpy.ndarray' object has no attribute 'keys'

And empty .csv file
so I think the installation is not really right

**Comment by Jonyian** [comment]
I also had the same problem

**Comment by bharath2438** [comment]
I also had the same issue and resolved it by doing the following https://github.com/flow-project/flow/issues/1071, it might work for you.
